mqemitter-redis.js

```diff
@@ -139,10 +140,11 @@ MQEmitterRedis.prototype.emit = function (msg, done) {
   }
 
   const p = this._pipeline.publish(msg.topic, msgpack.encode(packet))
+
   if (done) {
-    p.then(done, done)
+    p.then(() => done()).catch(done)
```

Comment:
Why is this needed?

```suggestion
    p.then(() => done(), done)
```

Reply:
`p.then(() => done()`

Is used because `p.then` returns an integer that is treated in done as an error with the previous signature.

Reply:
ok perfect.